### PR TITLE
Minor edits on README.md

### PR DESCRIPTION
**Pull request by Gudnessuche**
Be consistent in ending every sentence with a period (especially outside a parenthesis as in L16).

It keeps the document as perfectly professional as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Implementation details
   * Use a precomputed table of multiples of powers of 16 multiplied with the generator, so general multiplication becomes a series of additions.
   * Intended to be completely free of timing sidechannels for secret-key operations (on reasonable hardware/toolchains)
```
Comment (real-or-random) [inline README.md:58]: What about this line? And I think there are further lines.

**Review by real-or-random — COMMENTED** [review]
Perhaps it's easier to remove all periods in lists (except where a full sentence appears in a list item).

File: README.md [hunk 2]
```diff
@@ -32,18 +32,18 @@ Implementation details
...
-* Field operations
+  * Expose only higher level interfaces to minimize the API surface and improve application security. ("Be difficult to use insecurely").
+* Field operations.
   * Optimized implementation of arithmetic modulo the curve's field size (2^256 - 0x1000003D1).
-    * Using 5 52-bit limbs
+    * Using 5 52-bit limbs.
     * Using 10 26-bit limbs (including hand-optimized assembly for 32-bit ARM, by Wladimir J. van der Laan).
       * This is an experimental feature that has not received enough scrutiny to satisfy the standard of quality of this library but is made available for testing and review by the community.
-* Scalar operations
+* Scalar operations.
```
Comment (jonasnick) [inline README.md:41]: Agree with @real-or-random. Adding periods in places like this does not make sense.

Comment (Gudnessuche) [reply README.md:41]: While you're 100% correct that there might not be technical merit in these miniscule changes, it just irked me a little bit, that the words that described the technicals, weren't 100% punctuated, and as a grammar expert, I felt compelled to do the needful, for other grammarians in the future who'd probably come across this document.

Comment (jonasnick) [reply README.md:41]: Sorry, I don't get your comment. Closing this PR for now. Feel free to reopen if you have an updated suggestion.

**Pull request closed without merging**